Assets/Scripts/Control&Input/PlayerMovement.cs

```diff
@@ -132,15 +132,15 @@ private void SetCrouch(InputAction.CallbackContext ctx)
         {
             animator.SetBool("Crouching", true);
             strafeForce = strafeForceCrouched;
-            if (!IsInAir())
+            //if (!IsInAir())
                 inputManager.gameObject.LeanMoveLocalY(localCameraHeight - crouchedHeightOffset, 0.2f);
         }
             
         if (ctx.canceled)
         {
             animator.SetBool("Crouching", false);
             strafeForce = strafeForceGrounded;
-            if (!IsInAir())
+            //if (!IsInAir())
```

Comment:
What's going on with these commented-out checks that make execution always fall through to moving the camera?

Reply:
Forgot to remove properly, but the short story is that I found an undetected bug where players would be "locked" into crouching-heigth if crouch jumping and unpressing crouch while in air

The only reson we didn't see this bug earlier was that the crouching heigth was small enough for it not to be apparent

Reply:
Is now updated 👍